### PR TITLE
Feature/add edit page

### PR DESCRIPTION
**Pull request by Rhiana**
Adding in a demo edit page to test the UI
 - Extra info in the header
 - Translation descriptions and instructions

Bugfixes:
 - Gradient over the hero image on the homepage adjusted
 - Fixing sticky header
 - Increasing some font sizes

screen shot 2018-07-11 at 2 57 29 pm

screen shot 2018-07-11 at 4 10 05 pm

### CODE REVIEW

File: app/assets/stylesheets/base/settings_colours.scss [hunk 1]
```diff
@@ -66,3 +67,5 @@ $progress-background:     $light-grey-orange;
 $progress-edits:          $bright-yellow;
 $progress-awaiting:       $bright-orange;
 $progress-completed:      $mid-green;
+
+$transcript-background:   lighten($light-grey-orange, .2);
```
Comment (DanielHeath) [inline app/assets/stylesheets/base/settings_colours.scss]: Using `lighten` and similar quickly increases the total number of colors in your palette - use sparingly.

File: app/assets/stylesheets/components/transcript_info.scss [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  color: $body-foreground;
+  box-shadow: inset 0px -20px 20px 0px rgba(0,0,0,0.2);
+  padding: 30px;
+  &__content {
+    @include content_wrap;
+    display: grid;
+    grid-template-columns: 1fr 300px;
+    grid-template-areas:
+      "header header"
+      "info info"
```
Comment (DanielHeath) [inline app/assets/stylesheets/components/transcript_info.scss]: :heart: template areas

File: app/views/layouts/application_v2.html.erb [hunk 3]
```diff
@@ -3,6 +3,7 @@
   <head>
     <title>Amplify New Styling Demo</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css" integrity="sha384-lKuwvrZot6UHsBSfcMvOkWwlCMgc0TaWr+30HWe3a4ltaBwTZhyTEggF5tJv8tbt" crossorigin="anonymous">
```
Comment (DanielHeath) [inline app/views/layouts/application_v2.html.erb]: Can we host our assets ourselves instead of relying on the CDN?

Comment (Rhiana) [reply app/views/layouts/application_v2.html.erb]: Eventually yes, this is a fairly temp demo for the UI so setting this up quickly for now for approval

**Pull request merged**